### DISCUSSION TITLE
Does `WASM` work for front-end frameworks like vite, react, nextjs, etc 🤔 .

### DISCUSSION THREAD

**Discussion by jeet-dhandha** [tursodatabase/turso#521] [Q&A] [answered]
I m planning to build a front-end first application. So its mostly storing millions of rows, making aggregations on it, and many similar features.

**Comment by penberg** [comment, marked as answer]
Hey @jeet-dhandha, the `limbo-wasm` package only supports Node right now because there's no support for browser storage layer. I opened https://github.com/tursodatabase/limbo/issues/531 for this.

**Comment by LtdJorge** [comment]
@jeet-dhandha good news, the frontend support was merged! In case you don't know about it, you can also look into DuckDB, which is compatible with WASM too. DuckDB is for OLAP queries, while Limbo is for OLTP :)

**Comment by dikatok** [comment]
@penberg is there any existing opfs development?
